releases/example.yaml

```diff
@@ -12,9 +12,9 @@ release-notes:
     Features or settings that have been deprecated and need to be updated
     by the administrator.
 components:
-  admiral: <commit-hash/tag>
-  submariner: <commit-id>
-  lighthouse: <commit-id>
-  submariner-operator: <commit-id>
-  submariner-charts: <commit-id>
-  shipyard: <commit-hash/tag>
+  admiral: v0.5.0 # tag is possible
+  submariner: v0.5.0-rc0 # even pre-release tag
```

Comment:
how does using a tag work (outside of admiral and shipyard which need a small pre-release?) , ah, for a pre-release... ok :) that'd be reasonable..

Reply:
You'll be creating a pre release tag and using that for the admiral/shipyard versions.

Since our past releases didn't really follow this procedure, the example actually has to use the release tag, but for new releases the logic would work properly